File: beginner/__2000__/2930/2930.cpp
#include <iostream>
using namespace std;

int main(){

    int entrega, prazo;
    cin >> entrega >> prazo;

    if (prazo - entraga >= 1){

        cout << "Muito bem!, Apresenta antes do Natal!" << endl;
    }

    else if (prazo - entraga < 0){

        cout << "Eu te odeio professora!" << endl;
    }

    else if (prazo - entraga < 3){

        cout << "Parece o trabalho do meu filho!" << endl;
        prazo += 2;
        if (prazo <= 24){

            cout << "TCC Apresentado!" << endl;
        }
        else{

            cout << "Fail! Entao eh nataaaaal!" << endl;
        }
    }
    return 0;

}

File: beginner/__2000__/2930/2930.ts
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf8").trim().split(' ')
let [entrag, prazo ] = input.map(Number)

if (prazo - entraga >= 1){

    console.log("Muito bem!, Apresenta antes do Natal!")
}

else if (prazo - entraga < 0){

    console.log("Eu te odeio professora!")
}

else if (prazo - entraga < 3){

    console.log("Parece o trabalho do meu filho!")
    prazo += 2
    if (prazo <= 24){

        console.log("TCC Apresentado!")
    }
    else{

        console.log("Fail! Entao eh nataaaaal!")
    }
}